.github/workflows/release.yaml

```diff
@@ -0,0 +1,40 @@
+name: Release Action
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version (i.e. v1.2.3)'
+        required: true
+        type: string
+permissions:
+  contents: write
+jobs:
+  release:
+    name: Release Action
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/main' }}
+    steps:
+      - uses: actions/checkout@v3
+        name: Checkout Repository
+
+      - name: Set Node.js 16.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+
+      - name: Git config
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+
+      - name: Release version
+        run: npm version ${{ inputs.version }} -m "[AUTOMATED] Release version %s"
```

Comment:
Do you need a write access token for doing this? I think in the previous workflow that was the point of having secrets.VIB_ACTION_TOKEN

Reply:
Uhmm, I [tested](https://github.com/paleloser/vmware-image-builder-action/actions/runs/4303215635/jobs/7502671830) this in my own fork and worked, so I guess it should pass here too. Note that I added `contents: write` on the workflow `permissions`.

Reply:
The `--allow-same-version` seems useful, added in [75240d2](https://github.com/vmware-labs/vmware-image-builder-action/pull/140/commits/75240d2285a50dab8c42221a6ebef0b2d86dd9c8), thanks!